.github/BOTMETA.yml

```
@@ -469,6 +469,8 @@ files:
     maintainers: paginabianca
   $modules/database/misc/redis_data.py:
     maintainers: paginabianca
+  $modules/database/misc/redis_incr.py:
```

How about renaming it to `redis_data_incr`, since it's a bit like a side-care to `redis_data` and `redis_data_info`?
```suggestion
  $modules/database/misc/redis_data_incr.py:
```